src/04_output_statscan_census.R

```diff
@@ -0,0 +1,495 @@
+# this file is used for downloading data from StatsCan and other source. 
+# library("remotes")
+# install_github("bcgov/safepaths")
+pacman::p_load(cancensus,geojsonsf, tidyverse,config,bcmaps, bcdata, janitor,cansim,safepaths, arrow, duckdb)
+
+######################################################################################
+# 
+# 
+# Census Divisions (CD) are the principal geographic areas for the country-wide census and are created by Statistics
+# Canada. There are currently 28 CDs in B.C. (2011 Census). Census Division boundaries in Canada follow counties,
+# regional districts, regional municipalities, and five other types of geographic areas made up of groups of Census
+# Subdivisions (CSDs). In British Columbia the CD is comprised of one unorganized region (Stikine) and 27 Regional
+# Districts which have local government functions. CD boundaries follow Regional Districts so they are affected by
+# Regional District boundary changes, such as extensions, reductions, name changes, and newly created Regional
+# Districts.
+# Census Subdivisions (CSD) aggregate to Census Divisions (CDs), which in turn aggregate to a province or territory
+# (the province code for British Columbia is 59). This relationship is reflected in the 7-digit Standard Geographical
+# Classification code (SGC). The TMF SGC is slightly different from the Statistics Canada SGC due to the exclusion of the
+# province code. Therefor the TMF value for Census Division and Subdivision is referred to as the two digit CD code and
+# a three digit CSD code.
+# it is important to note that not all of the 837 CSD’s in the province will be represented on the TMF(Translation Master File which is the BC Stats official classification for geographic areas), as the appearance of
+# a CSD on the TMF is dependent upon whether or not there is a postal code geocoded to that area. A large number of
+# the RDEA’s and the vast majority of IRs are not represented on the TMF
+
+# Census Subdivision (CSD) CSDs are municipalities (as determined by provincial legislation) or areas treated as municipal equivalents 
+# 
+######################################################################################
+
+
+######################################################################################
+# Census data
+######################################################################################
+
+
+# the location is saved in config.yaml file. 
```

Comment:
Right. Saving these things in the local .Renviron is a solution. The only benefit from `config` package is if people don't want to mess with global .Renviron  and only want to a specific config for the project or want to get a project-specific config instead of a global .Renviron. If we can make a local .Renviron, it may be a better solution. Would like to know more about it.

Reply:
I didn't know much about .Renviron but this website explains it well:

https://www.roelpeters.be/what-is-the-renviron-file/

It's pretty easy to make local and global .Renviron files.